tests/unit/Framework/AssertTest.php

```diff
@@ -1524,6 +1524,19 @@ public function testAssertCountThrowsExceptionIfElementIsNotCountable(): void
         $this->fail();
     }
 
+    public function testInvalidArgumentExceptionUseCorrectArticleInErrorMessage(): void
```

Comment:
minor niggle: Uses instead of Use, third person singular

Reply:
> Uses instead of Use

Is it a requirement for all test methods? If yes, it'd be nice if `php-cs-fixer` will inform about issues like this. Or maybe a note should be added to the CONTRIBUTING.md?

Reply:
Thanks for bringing that up, I don't think the contribution guidelines are that detailed. For me this was also a nice-to-have, it doesn't affect the functionality.

At the next Code Sprint meetup early September I want to brainstorm ideas to make contributing easier.